.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: go
+sudo: required
+go:
+  - 1.12.x
```

Comment:
Would this work on older versions?  How far back can we go with the go modules?
Probably only 1.11 ?

Reply:
Go 1.11 is the first version to support modules.